devices/Button/Sample/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
```

Comment:
Even if it's the sample, please adjust this file as well

Reply:
thanks, fixed